### PR TITLE
Rocket

### PR DESCRIPTION
**Pull request by timotiousPrime**
added the rocket to the game

### CODE REVIEW

**Review by poltak — APPROVED** [review]
The code all looks ok. Though have we already decided how it fits in the 8-column play area? Can falling objects and the basket still go in front of the rocket, or the rocket should appear in its own special column?

The current implementation from these changes makes it so falling objects and the basket can go in front. Either is cool with me, but I just want to make sure we're all on the same page.
@tranhuuphucuong any opinions?

**Comment by tranhuuphucuong** [comment]
I think the rocket can be... not in another column inside `play-area` but next to it (details below), everything else doesn't have to change

File: index.html [hunk 1]
```diff
@@ -30,6 +30,9 @@
                     <li class="falling-object-column"></li>
                 </ul>
                 <img id="basket" class="basket" src="assets/basket.svg" />
+                <div class="rocketContainer">
+                    <img id="rocket" class="rocket" src="assets/rocket.svg" >
+                </div>
```
Comment (tranhuuphucuong) [inline index.html]: @timotiousPrime Just move the container one level out, same level with the `play-area`.
Makes it `position: relative`, same width as the rocket and height 100%,

File: stylesheet/main.css [hunk 2]
```diff
@@ -57,6 +58,24 @@ body {
...
+    width: 200px;
+    height: 250px;
+}
+
+.rocket {
+    z-index: 1;
+    position: absolute;
+    bottom: 0;
+    right: 5px;
+    width: 120px;
```
Comment (tranhuuphucuong) [inline stylesheet/main.css]: I think the rocket is too big, maybe `90px`?

**Comment by tranhuuphucuong** [comment]
@timotiousPrime Can you give it a try and see how it goes?

File: stylesheet/main.css [hunk 3]
```diff
@@ -57,6 +58,24 @@ body {
...
 }
 
+/* The rocket image won't display unless it is in a div */
+.rocketContainer {
+    position: absolute;
+    bottom: 15px;
+    right: 10px;
+    width: 200px;
+    height: 250px;
+}
```
Comment (tranhuuphucuong) [inline stylesheet/main.css]: Also, when you change position to relative, you also want to remove the `bottom` and `right` props

Comment (timotiousPrime) [reply stylesheet/main.css]: I'll give it a go

Comment (timotiousPrime) [reply stylesheet/main.css]: I've done these changes, check and let me know. Bev is going to be adding a longer rocket that I will replace this one once it's done

**Review by tranhuuphucuong — APPROVED** [review]
LGTM!!.... great work TIm

**Pull request merged**